src/main.ts

```diff
@@ -79,6 +89,38 @@ export default class ShikiPlugin extends Plugin {
 		await this.registerPrismPlugin();
 	}
 
+	async loadCustomThemes(): Promise<void> {
+
+		// @ts-expect-error TS2339
+		const themePath = this.app.vault.adapter.path.join(this.app.vault.configDir, 'plugins', this.manifest.id, 'themes');
```

Comment:
It would be better to extend types via a `obsidian-ex.d.ts` file than to use `@ts-expect-error`